ecl/hql/hqllex.l

```diff
@@ -871,6 +871,7 @@ RECORD              { RETURNHARD(RECORD); }
 RECORDOF            { RETURNSYM(RECORDOF); }
 RECOVERY            { RETURNSYM(RECOVERY); }
 REGEXFIND           { RETURNSYM(REGEXFIND); }
+REGEXFINDSET        { RETURNSYM(REGEXFINDSET); }
```

Comment:
In line 10711, I have: 
    case REGEXFINDSET: msg.append("REGEXFINDSET"); break;

Isn't that what you were suggesting?  (By the way, eclcc doesn't report any more errors after I added this entry.)